src/persistence.ts

```diff
@@ -48,22 +48,22 @@ export interface ConditionalDeleteResourceRequest {
 
 export interface InitiateExportRequest {

```

Comment:
I think it may make more sense to have the Export and Import separate components, similar to what we did with bundles - thoughts?

Reply:
Making it separate components